omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . '/omise/classes/charge.php';
+}
+
```

Comment:
@guzzilar Now, the discussion is should `OmisePaymentModuleFrontController` class send the parameter to `Charge.create()`?

From the above example:

- The coupling between `OmisePaymentModuleFrontController ` and `Charge` is increased with an array parameter that contained 4 parameters inside.

- The dependency of `OmisePaymentModuleFrontController ` is increased more 3 (`Tools`, `Cart` and `Currency`). Which it return to the original problem that required refactoring.

- The parameter that used to send to Omise API has been separated to 2 places. Some in `OmisePaymentModuleFrontController` and some in `Charge`.

- If the parameter that need to send to Omise API has been changed, it will impact through to the controller class. It is not be scoped in one class or one place.